lib/discordrb/data/channel.rb

```diff
@@ -979,6 +993,14 @@ def remove_recipient(recipient)
       @recipients.delete(recipient)
     end
 
+    # Set the last pin timestamp of a channel.
+    # @param time [String, nil] the time of the last pinned message in the channel
+    # @note For internal use only
+    # @!visibility private
+    def process_last_pin_timestamp(time)
+      @last_pin_timestamp = time.is_a?(String) ? Time.parse(time) : time
```

Comment:
if we're going to do a ternary do we need to do a string check? `time ? Time.parse(time) : nil`. nitpicky though